cibyl/models/openstack/deployment.py

```diff
@@ -0,0 +1,63 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+
+from cibyl.cli.argument import Argument
+from cibyl.models.model import Model
+from cibyl.models.attribute import AttributeListValue
+
+
+class Deployment(Model):
+    """Openstack deployment model"""
+
+    API = { 
+        'release': {
+            'attr_type': float,
+            'arguments': [Argument(name='--release', arg_type=float,
+                description="Deployment release number")]
+        },                                                                                        
+        'infra_type': {
+            'attr_type': str,
+            'arguments': [Argument(name='--infra-type', arg_type=str,
+                description="Infra type")]
+        },
+        'nodes': {
+            'attr_type': str,
+            'attribute_value_class': AttributeListValue,
+            'arguments': [Argument(name='--nodes', arg_type=str,
```

Comment:
arg_type=Node

You'll probably need to add an import like that: from cibyl.models.openstack.node import Node